gradio/components.py

```diff
@@ -277,6 +277,7 @@ def __init__(
         self.max_lines = max_lines
         self.placeholder = placeholder
         self.default_value = default_value
+        self.cleared_value = ""
```

Comment:
Should `cleared_value` be a property in the backend? I feel like it would be cleaner if it was defined as a prop in the frontend.